gpt_index/indices/composability/utils.py

```diff
@@ -0,0 +1,63 @@
+from typing import Any, Dict, Optional
+from gpt_index.constants import VECTOR_STORE_KEY
+from gpt_index.vector_stores.registry import (
+    load_vector_store_from_dict,
+    save_vector_store_to_dict,
+)
+from gpt_index.vector_stores.types import VectorStore
+
+
+def save_query_context_to_dict(
+    query_context: Dict[str, Dict[str, Any]]
+) -> Dict[str, Dict[str, Any]]:
+    """Save index-specific query context dict to JSON dict.
+
+    Example query context dict to save:
+    query_context = {
+        index_id: {
+            'vector_store': vector_store
```

Comment:
this docstring isn't actually accurate right?

Reply:
I'm just using <index_id> to represent the str, and <vector_store> to represent the vector store instance.

Reply:
I'll revise to be less confusing.